src/main/java/com/sonymobile/jenkins/plugins/lenientshutdown/ShutdownRunListener.java

```diff
@@ -60,28 +60,26 @@ public void onCompleted(final R r, TaskListener listener) {
         Executor executor = r.getExecutor();
         if (executor != null) {
             final Computer computer = executor.getOwner();
-            if (computer != null) {
-                final String nodeName = computer.getName();
+            final String nodeName = computer.getName();
```

Comment:
this too?

Reply:
[line 575 of core/src/main/java/hudson/model/Computer.java at jenkins-2.504.3 in jenkinsci/jenkins](https://github.com/jenkinsci/jenkins/blob/jenkins-2.504.3/core/src/main/java/hudson/model/Computer.java#L575) shows us that `Computer#getName()` is annotated with `@NonNull`.